cpg_workflows/status.py

```diff
@@ -35,12 +35,10 @@ def complete_analysis_job(
         update_analysis_meta (Callable | None): function to update analysis meta
         tolerate_missing (bool): if True, allow missing output
     """
-    import traceback
 
     from cpg_utils import to_path
-    from metamist.apis import AnalysisApi
-    from metamist.exceptions import ApiException
-    from metamist.models import Analysis, AnalysisStatus
+
+    from .metamist import AnalysisStatus, get_metamist
```

Comment:
We just need to keep an eye on this it case it creates any pickling issues. Let's try a test run later!